sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobServiceSasSignatureValues.java

```diff
@@ -358,6 +370,7 @@ public String getIdentifier() {
      * @param identifier Name of the access policy
      * @return the updated BlobServiceSASSignatureValues object
      */
+    @Deprecated
```

Comment:
We don't want to deprecate this because a customer may use the expiry + permissions overload but still want to set an identifier to coordinate start times or for revoke purposes.

Reply:
That sounds reasonable, will undo. 

Though why couldnt a customer just use the constructor for identifier and then set the expiryTime and permissions with the setters?

Reply:
They could, but you could use that argument in reverse, right? You could deprecate expiry and permissions, and then customers could always set those in the constructor and do everything else via setters. The point of the constructors is to hopefully lead people to understand a minimal set of required parameters, but we also need to enable them to use setters because there's actually not a single parameter that strictly *has* to be in the query string